src/openrct2/platform/platform.h

```diff
@@ -35,6 +35,10 @@
 #define MAX_PATH 260
 #endif
 
+#ifndef MAX_COMMAND_LENGTH
+#define MAX_COMMAND_LENGTH 512
```

Comment:
It was my idea to use 512. I'd say that that is enough. It's nice that we could use 2M of arguments, but I doubt it's worth the memory.